Src/BootCamp.Chapter/Checks.cs

```diff
@@ -65,138 +50,122 @@ public static string FindRichestPerson(string[] peopleAndBalances)
             }
-            StringBuilder biggestLossName = new StringBuilder();
+            StringBuilder name = new StringBuilder();
             decimal amount = 0;
             int personCount = 1;
 
             for (int i = 0; i < peopleAndBalances.Length; i++)
             {
-                var account = new Account(MakeStuffWithInput.GetNameFromString(peopleAndBalances[i]), MakeStuffWithInput.GetBalanceFromString(peopleAndBalances[i]));
+                Account account = AccountCreator(peopleAndBalances[i]);
 
-                if (account.MoreThan1Balance())
+                if (account.IsLessThan2Balance())
                 {
 
                 }
-                else if (biggestLossName.ToString() == "")
+                else if (name.ToString() == "")
                 {
-                    biggestLossName.Append(account.GetName());
+                    name.Append(account.GetName());
                     amount = account.BiggestLoss();
                 }
                 else if (amount == account.BiggestLoss())
                 {
-                    if (personCount == 1)
-                    {
-                        biggestLossName.Append(" and " + account.GetName());
-                        personCount++;
-                    }
-                    else
-                    {
-                        biggestLossName.Replace(" and ", ", ");
-                        biggestLossName.Append(" and " + account.GetName());
-                        personCount++;
-                    }
+                    personCount = AddPersonToExising(name, personCount, account);
                 }
                 else if (amount > account.BiggestLoss())
                 {
-                    biggestLossName.Clear();
-                    biggestLossName.Append(account.GetName());
+                    name.Clear();
+                    name.Append(account.GetName());
                     amount = account.BiggestLoss();
                     personCount = 1;
                 }
             }
 
-            return MakeStuffWithInput.MakefullstringBiggestLossPeople(biggestLossName.ToString(), amount);
+            return MakeStuffWithInput.MakefullstringBiggestLossPeople(name.ToString(), amount);
         }
 
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
             if (Testers.IsThisStringArrayValid(peopleAndBalances))
             {
-                return "N/A.";
+                return nA;
             }
-            StringBuilder richestName = new StringBuilder();
+            StringBuilder name = new StringBuilder();
             decimal amount = 0;
             int personCount = 1;
 
             for (int i = 0; i < peopleAndBalances.Length; i++)
             {
-                var account = new Account(MakeStuffWithInput.GetNameFromString(peopleAndBalances[i]), MakeStuffWithInput.GetBalanceFromString(peopleAndBalances[i]));
+                Account account = AccountCreator(peopleAndBalances[i]);
```

Comment:
AccountCreator is an account? I think it should simply be renamed to CreateAccount

Reply:
Or even better- ParseAccount